engine/apps/phone_notifications/models/sms.py

```diff
@@ -45,7 +45,7 @@ class Meta:
     notification_policy = models.ForeignKey(
         "base.UserNotificationPolicy", on_delete=models.SET_NULL, null=True, default=None
     )
-
+    represents_bundle_uuid = models.CharField(max_length=100, null=True, default=None, db_index=True)
```

Comment:
can this be an optional foreign key to a `BundledNotification` object instead?

Reply:
it represents a bunch of `BundledNotification`s, so it should be `ManyToMany`, what means +1 db table. But since we already have a unique identifier for `BundledNotification`, we can avoid adding additional many-to-many table and use this identifier instead 🤔